frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx

```diff
@@ -135,56 +137,66 @@ export const AddCapacityModal = (props: AddCapacityModalProps) => {
+          fieldId="add-cap-sc-dropdown__FormGroup"
+          isRequired
+        >
+          <Title className="pf-u-pb-sm" headingLevel="h5">
+            {t('ceph-storage-plugin~Storage Class')}&nbsp;&nbsp;
+            <Tooltip position="top" content={<>{storageClassTooltip(t)}</>}>
+              <QuestionCircleIcon className="pf-u-secondary-color-100" />
+            </Tooltip>
+          </Title>
+          <div className="ceph-add-capacity__sc-dropdown">
             <OCSStorageClassDropdown onChange={onChange} data-test="add-cap-sc-dropdown" />
           </div>
-          {isNoProvionerSC ? (
-            <PVsAvailableCapacity
-              replica={replica}
-              data-test-id="ceph-add-capacity-pvs-available-capacity"
-              storageClass={storageClass}
-            />
-          ) : (
-            <div>
-              <label className="control-label" htmlFor="requestSize">
-                {t('ceph-storage-plugin~Raw Capacity')}
-                <FieldLevelHelp>{requestedCapacityTooltip}</FieldLevelHelp>
-              </label>
-              <div className="ceph-add-capacity__form">
-                <input
-                  className={classNames('pf-c-form-control', 'ceph-add-capacity__input')}
-                  type="number"
-                  name="requestSize"
-                  value={osdSizeWithoutUnit}
-                  required
-                  disabled
-                  data-test-id="requestSize"
-                />
-                {provisionedCapacity && (
-                  <div className="ceph-add-capacity__input--info-text">
-                    {t('ceph-storage-plugin~x {{ replica, number }} replicas =', { replica })}{' '}
-                    <strong data-test="provisioned-capacity">{provisionedCapacity} TiB</strong>
-                  </div>
-                )}
-              </div>
-              <div className="ceph-add-capacity__current-capacity">
-                <div className="text-secondary ceph-add-capacity__current-capacity--text">
-                  <strong>{t('ceph-storage-plugin~Currently Used:')}</strong>
-                </div>
+        </FormGroup>
+        {isNoProvionerSC ? (
+          <PVsAvailableCapacity
+            replica={replica}
+            data-test-id="ceph-add-capacity-pvs-available-capacity"
+            storageClass={storageClass}
+          />
+        ) : (
+          <>
+            <FormGroup
+              className="pf-u-py-sm"
+              fieldId="requestSize__FormGroup"
+              id="requestSize__FormGroup"
+            >
+              <Title className="pf-u-pb-sm" headingLevel="h5">
+                {t('ceph-storage-plugin~Raw Capacity')}&nbsp;&nbsp;
+                <Tooltip position="top" content={<>{requestedCapacityTooltip(t)}</>}>
+                  <QuestionCircleIcon className="pf-u-secondary-color-100" />
+                </Tooltip>
+              </Title>
+              <TextInput
+                isDisabled
```

Comment:
Whenever you add a child to `FormGroup` , make sure to add the the `id` to chld with value being same as `fieldId`.
It connects them semantically and hence a`required` prop.